lib/internal/debugger/inspect_client.js

```diff
@@ -297,7 +297,8 @@ class Client extends EventEmitter {
 
   async _discoverWebsocketPath() {
     const { 0: { webSocketDebuggerUrl } } = await this._fetchJSON('/json');
-    return URL.parse(webSocketDebuggerUrl).path;
+    const { pathname, search } = new URL(webSocketDebuggerUrl);
+    return `${pathname}${search}`;
```

Comment:
I believe this needs to be in a different pull request, since the title does not reflect this change.

Reply:
Would making it into two commits ("use `internal/url` in `lib`" with code changes and "restrict internal code from using public `url`" with eslint change) be acceptable?

Reply:
Yes 👍

Reply:
Force-pushed on top of just-in-time https://github.com/nodejs/node/pull/49558 😄